keras_hub/src/models/qwen/qwen_attention.py

```diff
@@ -0,0 +1,96 @@
+import keras
+from keras import ops
+
+from keras_hub.src.models.llama.llama_attention import LlamaAttention
+from keras_hub.src.utils.keras_utils import has_flash_attention_support
+
+
+class Qwen2Attention(LlamaAttention):
```

Comment:
It's always a good idea to write in the doc-string how the subclass is different from the superclass, whenever you are inheriting.

Reply:
@divyashreepathihalli - are we okay with cross-model inheriting like this?

Reply:
Yeah lets keep them separate and copy the code over.

Reply:
Alright done!